### ISSUE TITLE
Regression Tests producing different data

### ISSUE THREAD

**Issue by TSGames** [drewnoakes/metadata-extractor#626]
When running the regression test (Windows 11 JVM, German OS + German Timezone) I'm getting a diff in multiple files, especially comments or special data. It looks like an issue with file encoding, but I can't figure out what is causing it.

Some examples:
![image](https://github.com/drewnoakes/metadata-extractor/assets/12344992/9bc7bddc-06d7-49cc-a9eb-3ecefaddd034)
![image](https://github.com/drewnoakes/metadata-extractor/assets/12344992/57b11622-fe73-4395-99f0-a21a99b7581d)

Any help/assistance would be appreciated!

**Comment by drewnoakes** [comment]
> For Java, perhaps we need something like Locale.setDefault(...) to force a particular culture for consistency across those files.

Ug I hadn't had my coffee --that's exactly what your PR that I approved yesterday does :)

**Comment by TSGames** [comment]
Yes, that solved the issues regarding number formatting. I tried setting default encoding to utf 8 but that doesn't made a difference (I guess that is default on java anyways)

Can you tell me on which is and locale you're running these tests? If nothing helps I would configure an environment m

**Comment by drewnoakes** [comment]
I'm on Windows 11, running on en-AU.

I've made some progress on date formatting, by converting everything to UTC in the output files and will push that up later today hopefully.

Were you seeing diffs in the .NET files or the Java ones?

**Comment by drewnoakes** [comment]
BTW you can workaround the issue here by running the suite without your changes and staging the diff, then running with your changes and comparing the workspace with the index.

**Comment by TSGames** [comment]
> I'm on Windows 11, running on en-AU.

Alright, I just wanted to make sure it has nothing to Do with linux/windows encoding.

Will then probably go the git staging route to compare the cr3 PR changes.

**Comment by drewnoakes** [comment]
That probably makes the most sense for now. I can re-run your branch against the regression set too, as an extra check.
